scripts/upgrade

```diff
@@ -164,7 +171,34 @@ ynh_replace_string  --match_string="yunobase"   --replace_string="$db_name"    -
 ynh_replace_string  --match_string="yunomail"   --replace_string="$email"      --target_file="$config"
 ynh_replace_string  --match_string="yunodomain" --replace_string="$domain"     --target_file="$config"
 ynh_replace_string  --match_string="language"   --replace_string="$language"   --target_file="$config"
-ynh_replace_string  --match_string="__DOMAIN__" --replace_string="$domain" --target_file="$config"
+ynh_replace_string  --match_string="__DOMAIN__" --replace_string="$domain"     --target_file="$config"
+# Enable or disable DAV support for users
+if [ $dav_support -eq 0 ]
+then
+	ynh_replace_string --match_string="__DAV__" --replace_string="false" --target_file="$config"
+else
+	ynh_replace_string --match_string="__DAV__" --replace_string="true" --target_file="$config"
+	dav_support=1
+	ynh_app_setting_set --app=$app --key=dav_support --value=$dav_support
+fi
+# Enable or disable signup for public users
+if [ $signup -eq 1 ]
+then
+	ynh_replace_string --match_string="__SIGNUP__" --replace_string="false" --target_file="$config"
+else
+	ynh_replace_string --match_string="__SIGNUP__" --replace_string="true" --target_file="$config"
+	signup=0
+	ynh_app_setting_set --app=$app --key=signup --value=$signup
+fi
+# Enable or disable two factor authentication support for  users
+if [ $two_factor -eq 1 ]
+then
+	ynh_replace_string --match_string="__TWO_FACTOR__" --replace_string="true" --target_file="$config"
+else
+	ynh_replace_string --match_string="__TWO_FACTOR__" --replace_string="false" --target_file="$config"
+	two_factor=0
+	ynh_app_setting_set --app=$app --key=two_factor --value=$two_factor
+fi
```

Comment:
Same stuff as the install, could be heavily simplified using ynh_add_config

Reply:
done